packages/@react-spectrum/list/docs/ListView.mdx

```diff
@@ -0,0 +1,345 @@
+    Row with action menu
+    <ActionMenu>
+      <Item key="edit" textValue="Edit">
+        <Edit />
+        <Text>Edit</Text>
+      </Item>
+      <Item key="delete" textValue="Delete">
+        <Delete />
+        <Text>Delete</Text>
+      </Item>
+    </ActionMenu>
+  </Item>
+</ListView>
+```
+
+## Selection
+
+By default, ListView doesn't allow row selection but this can be enabled using the `selectionMode` prop.
+Use `defaultSelectedKeys` to provide a default set of selected rows. Note that the value of the selected keys must match the `key` prop of the Item.
+
+The example below enables multiple selection mode, and uses `defaultSelectedKeys` to select the rows with keys "row1" and "row3".
+
+```tsx example
+<ListView selectionMode="multiple" defaultSelectedKeys={["row1", "row3"]} aria-label="List seletion example">
+  <Item key="row1" textValue="row 1">
+    <Content>row 1</Content>
+  </Item>
+  <Item key="row2" textValue="row 2">
+    <Content>row 2</Content>
+  </Item>
+  <Item key="row3" textValue="row 3">
+    <Content>row 3</Content>
+  </Item>
+</ListView>
+```
+
+### Controlled selection
+
+To programmatically control row selection, use the `selectedKeys` prop paired with the `onSelectionChange` callback. The `key` prop from the selected rows will
+be passed into the callback when the row is pressed, allowing you to update state accordingly.
+
+Here is how you would control selection for the above example.
+
+```tsx example export=true
+function PokemonList(props) {
+  let rows = [
+    {id: 1, name: 'Charizard'},
+    {id: 2, name: 'Blastoise'},
+    {id: 3, name: 'Venusaur'},
+    {id: 4, name: 'Pikachu'}
+  ];
+
+  let [selectedKeys, setSelectedKeys] = React.useState(new Set([2]));
+
+  return (
+    <ListView items={rows} aria-label="List with controlled selection" selectionMode="multiple" selectedKeys={selectedKeys} onSelectionChange={setSelectedKeys} {...props}>
+      {(item) => (
+        <Item key={item.id} textValue={item.name}>
+          {item.name}
+        </Item>
+      )}
+    </ListView>
+  );
+}
+```
+
+### Single selection
+
+To limit users to selecting only a single item at a time, `selectionMode` can be set to `single`.
+
+```tsx example
+// Using the same list as above
+<PokemonList selectionMode="single" />
+```
+
+### Disallow empty selection
+
+ListView also supports a `disallowEmptySelection` prop which forces the user to have at least one row in the ListView selected at all times.
+In this mode, if a single row is selected and the user presses it, it will not be deselected.
+
+```tsx example
+// Using the same list as above
+<PokemonList disallowEmptySelection />
+```
+
+### Disabled rows
+
+You can disable specific rows by providing an array of keys to ListView via the `disabledKeys` prop. This will prevent rows from being selectable as shown in the example below.
+
+```tsx example
+// Using the same list as above
+<PokemonList selectionMode="multiple" disabledKeys={[3]} />
```

Comment:
`disabledKeys` isn't working here, looking into this.

Reply:
Yep, tested it locally, removing the `key={item.id}` from the PokemonList `Item` fixes it

Reply:
Good catch, just pushed that fix